pandas/tests/plotting/common.py

```diff
@@ -53,10 +53,7 @@ def setup_method(self, method):
         import matplotlib as mpl
         mpl.rcdefaults()
 
-        self.mpl_ge_2_0_1 = plotting._compat._mpl_ge_2_0_1()
-        self.mpl_ge_2_1_0 = plotting._compat._mpl_ge_2_1_0()
-        self.mpl_ge_2_2_0 = plotting._compat._mpl_ge_2_2_0()
-        self.mpl_ge_2_2_2 = plotting._compat._mpl_ge_2_2_2()
+        self.mpl_eq_2_2_2 = plotting._compat._mpl_eq_2_2_2()
```

Comment:
shouldn't you just make this .mpl_ge_2_2_2

Reply:
It was intentional because the changes in `test/plotting/test_datetimelike` affected *only* 2.2.2, but after the bump to 2.2.2, this can now be formulated also with `_ge_`.